ec2-spot-elastic-inference/EI_launch_template.json

```diff
@@ -1,12 +1,12 @@
 {
-  "LaunchTemplateData": {
+  
     "EbsOptimized": true,
     "IamInstanceProfile": {
-      "Arn": "%instanceProfile%"
+      "Arn": "arn:aws:iam::000474600478:instance-profile/EI-Spot"
```

Comment:
This is specific to your account, we should keep the placeholders. Please don't change those